package.json

```diff
@@ -36,7 +41,12 @@
     "lodash.get": "^4.4.2",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "rx-lite": "^4.0.8"
+    "rxjs": "^6.3.3"
+  },
+  "peerDependencies": {
+    "react": "16.x",
```

Comment:
Shouldn't we remove react from "dependencies" then?

Reply:
@McReader, no. If we remove it from "dependencies" it'll break.
Since npm v3, "peerDependencies" don't install anything, but looks for a dependency in node_modules and throws a warning if it's not found.

Reply:
@McReader, by the way, I've just got a feeling that we should move a bunch of our "devDependencies" (such as all the babel stuff) into "peerDependencies". Reasons: different versions of babel stuff and especially, I doubt that the library will work without "@babel/plugin-proposal-decorators" installed.
"@babel/plugin-proposal-decorators" takes part directly in out public API, thus, we need to require it to be installed in the project's dependencies.

Reply:
Need to discuss it face 2 face. We have a lot of thinks to discuss related to dependencies

Reply:
Will be discussed and changed according to [this](https://github.com/McReader/observe-props-decorator/issues/10) issue. We need to merge this MR, since it's blocker for [this task](https://github.com/McReader/observe-props-decorator/issues/2)